__init__.py

```diff
@@ -201,3 +201,18 @@ def __init__(self, opsdroid, config):
         vm_name = message.regex.group('vm_name')
         live_mount = rubrik.get_vsphere_live_mount_names(vm_name)
         await message.respond('All done! {} has the following live mounts: {}.'.format(vm_name, live_mount))
+
+    @match_regex('get live mounts of sql db (?P<db_name>[\w\'-]+) on (?P<sql_instance>[\w-]+) on host (?P<sql_host>.+)')
+    async def getsqllivemount(self, message):
+        """
+        A skills function to list the current live mounts of a MSSQL database. The parser looks for the message argument.
```

Comment:
change to `A skills function to list the current live mounts of an MSSQL database. The parser looks for the message argument.`